### PR TITLE
ADEN-11921 Reconfigure Eyeota integration for GDPR etc

### PR DESCRIPTION
**Pull request by szymon-nawrocki**
### Ticket
https://fandom.atlassian.net/browse/ADEN-11921

### CODE REVIEW

File: spec/ad-services/eyeota/eyeota.spec.ts [hunk 1]
```diff
@@ -15,30 +15,62 @@ describe('Eyeota', () => {
...
 
 	afterEach(() => {
 		sandbox.restore();
 		context.remove('services.eyeota.enabled');
 		delete window.__tcfapi;
-	});
 
-	it('is called when enabled in the context', async () => {
-		context.set('services.eyeota.enabled', true);
+		context.set('services.eyeota.enabled', undefined);
```
Comment (preludek) [inline spec/ad-services/eyeota/eyeota.spec.ts:30]: I'd use `context.remove` here. Or bonus points for adding a `clear` method or something like that, that removes all at once.

File: src/communication/event-types.ts [hunk 2]
```diff
@@ -178,6 +178,9 @@ export const eventsRepository: Dictionary<EventOptions> = {
 	AUDIGENT_SEGMENTS_READY: {
 		name: 'Audigent segments ready',
 	},
+	EYEOTA_STARTED: {
```
Comment (preludek) [inline src/communication/event-types.ts:181]: semantics but maybe `ready`? ;)

**Review by preludek — APPROVED** [review]

**Review by preludek — CHANGES_REQUESTED** [review]
wrong button xd

File: src/communication/event-types.ts (hunk 2 above)
Comment (szymon-nawrocki) [reply src/communication/event-types.ts:181]: I used `start` to be consistent with the events defined here:
https://github.com/Wikia/ad-engine/blob/289d98e9fd4e60fdcf4c1336ee7d4738be80228c/src/ad-engine/services/load-times-service.ts#L6

Plus it is clearly stated in the ticket to add "integration_**start**" event :)

**Review by preludek — APPROVED** [review]

**Pull request merged**